addons/website_event/controllers/main.py

```diff
@@ -42,10 +42,14 @@ def _get_events_search_options(self, **post):
 
     @http.route(['/event', '/event/page/<int:page>', '/events', '/events/page/<int:page>'], type='http', auth="public", website=True, sitemap=sitemap_event)
     def events(self, page=1, **searches):
-        if searches.get('tags') and request.httprequest.method == 'GET' and not searches.get('prevent_redirect'):
+        if len(searches.get('tags', '[]')) > 3 and request.httprequest.method == 'GET' and not searches.get('prevent_redirect'):
             # Previously, the tags were searched using GET, which caused issues with crawlers (too many hits)
             # We replaced those with POST to avoid that, but it's not sufficient as bots "remember" crawled pages for a while
             # This permanent redirect is placed to instruct the bots that this page is no longer valid
+            # Note: We allow a single tag to be GET, to keep crawlers & indexes on those pages
+            # What we really want to avoid is combinatorial explosions
+            # As 'tags' contains an array repr as a string, we need to test that it's greater than 3 (str length)
+            # '[]' is allowed, '[5]' is allowed, '[5,6]' is not
```

Comment:
And `[42]`?

Reply:
Waw... good job me, I really need some vacation.

Here is a cat to make you forget this ever happened:
![image](https://github.com/user-attachments/assets/955dea9f-c17d-4b8f-bbd1-6276d83f94c0)

Thanks for the heads-up and the suggestion ^^